src/debug/seer-integration.js

```diff
@@ -10,6 +12,54 @@ export const addModel = model => {
     return;
   }
   models[model.id] = model;
+
+  seer.listItem('luma.gl', model.id);
+};
+
+const getType = obj => Object.prototype.toString.call(obj).slice(8, -1);
+
+const getAttribute = (key, attr) => {
+  const isArray = getType(attr).includes('Array');
+  if (isArray) {
+    return Array.prototype.slice.call(attr);
```

Comment:
Again explain why not `attr.slice()` to the rest of us who are not JS one liner fiends.